Training/Training/SortAndSearch.c

```diff
@@ -0,0 +1,46 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Gokul TS @Metamation India.
+// ------------------------------------------------------------------
+// SortAndSearch.c
+// C Program for Bubble sort,Insertion sort and binary search algorithm
+// ------------------------------------------------------------------------------------------------
+#include "SortAndSearch.h"
+
+void BubbleSort (int arr[], int size) {
+   int swapped;

```

Comment:
You can move the swapped inside the for loop to limit the scope of the variable. So, its declaration and initialization can be done inside itself.